hw_4.js

```diff
@@ -0,0 +1,365 @@
+"use strict"
+
+// Task 1
+// Написать функцию getSum, которая будет высчитывать сумму чисел от нуля, до параметра, 
+// который мы в неё передаем.
+
+// let X = prompt('Введите целое положительное число X: ','');
+// function getSum () {
+//    let result = 0;
+//    for (let a = 0; a < X; a++) {
+//       result += a + 1;
+//    }
+//       return result;
+// }
+//       alert(`Сумма всех чисел от 0 до X:  ${getSum ()}`);
+
+
+// Task 2
+// Напишите функцию которая в качестве аргумента принимает в себя сумму кредита, 
+// который хочет получить клиент и верните результат переплаты по кредиту:
+// процентная ставка в год — 17%, количество лет — 5, сумма кредита не увеличивается
+
+// let creditAmount = prompt('Введите сумму кредита, BYN', "");
+// let creditRate = prompt('Введите процентную ставку, %', "");
+// let yearsNumber = prompt('Введите срок кредита, лет', "");
+
+// function overPayment() {
+//    let over = creditAmount * creditRate / 100 * 5;
+//    return over;
+// }
+//    alert(`Сумма переплаты составит, BYN: ${overPayment()}`);
+
+
+   // Task 3
+   // Написать функцию trimString которая в качестве аргумента принимает три параметра: 
+   // строку, значение от, значение по.
+   // После вызова функция должна вернуть переданную строку обрезанную по значениям от и по.
+
+//    let line = prompt('Введите строку', "");
+//    let cutFrom = prompt('Обрезать строку от символа №', "");
+//    let cutTill = prompt('Обрезать строку до символа №', "");
+//    console.log('Исходная строка:',line);
+//    console.log('Обрезать от символа',cutFrom);
+//    console.log('Обрезать до символа',cutTill);
+
+// function trimString () {
+//    let newLine = line.slice((cutFrom - 1), cutTill);
+//    return newLine;
+// }
+// console.log('Обрезанная строка:',trimString());
+
+
+   // Task 4
+   // Написать функцию getSumNumbers, которая будет принимать число 
+   // и вычислять сумму цифр из которых состоит число.
+
+// let numberIn = prompt('Введите число',"");
+// let summ = 0;
+// function getSumNumbers () {
+//       let arr = Array.from('' + numberIn);
+//       for (let number of arr.map(Number)) {
+//       summ += number;
+//       }
+//       return summ;
+// }
+// alert(getSumNumbers ());
+
+
+// Task 5
+// Написать функцию getSum которая принимает два целых числа a и b, 
+// которые могут быть положительными или отрицательными, найти сумму всех чисел между ними, 
+// включая их, и вернуть ее. Если два числа равны, верните a или b.
+
+// let X = +prompt('Введите 1е число', '');
+// let Y = +prompt('Введите 2е число', '');
+// let Sum = 0;
+// if (X < Y) {
+//    } else {
+//    [X, Y] = [Y, X];
+// }
```

Comment:
зачем пустой блок при true? почему не сделать так?
```suggestion
// if (X > Y) {
//    [X, Y] = [Y, X];
// }
```